### PR TITLE
WIP: Update console.md for tone/grammar/verbiage

### PR DESCRIPTION
**Pull request by celanthe**
WIP: Updating the console docs for tone/grammar/verbiage and clarity.

### CODE REVIEW

File: docs/home/console.md [hunk 1]
```diff
@@ -114,38 +113,35 @@ Select the new user’s role from the dropdown menu.
...
+If you’d like to simulate an Android device, click **Simulare Device**. A notification will appear, displaying a success message with the text, "You are all set!" A mock device will then be available for you to experiment with Esper to and learn how Esper interacts with devices. Feel free to make use of the Simulare until you’re ready to provision an actual device.
 
-Capabilities such as Apps, Remote View and Capture log may not be available on a Simulated device.
+Capabilities such as Apps, Remote View and Capture log may not be available on a simulated device.
 
-:::tip
-This feature is only available on accounts created after Friday, September 27, 2019. If you became an Esper client before then and would like access to a simulated device, [contact us](mailto:support@esper.io).
-:::
+:::tip The Simulare feature is only available on accounts created after Friday, September 27, 2019. If you became an Esper client before then and would like access to a simulated device, [contact us](mailto:support@esper.io). :::
```
Comment (QueenOvDroids) [inline docs/home/console.md:144]: Simulare's are only available as a one-time provisioning option for newly-created accounts, actually - verified in Keith's Getting Started update. May be worth clarifying.

File: docs/home/console.md [hunk 2]
```diff
@@ -114,38 +113,35 @@ Select the new user’s role from the dropdown menu.
 If you’re creating a Group Admin or Group Viewer, assign to the appropriate groups and click **Confirm**.
 ![Add GroupUser](./assets/NewConsole/UserAddGroup.jpg)
 
-After adding a user, be sure to give the user their login credentials so that they can log in.
+After adding a user, be sure to provide the user with their login credentials so that they can succesfully log in.
```
Comment (QueenOvDroids) [inline docs/home/console.md:116]: successfully - second s

**Review by QueenOvDroids — APPROVED** [review]
Fantastic work.

File: docs/home/console.md (hunk 2 above)
Comment (celanthe) [reply docs/home/console.md]: Made a new PR correcting this typo! :) Thanks for pointing it out!

**Review by QueenOvDroids — APPROVED** [review]

**Review by kirananto — APPROVED** [review]

**Pull request merged**